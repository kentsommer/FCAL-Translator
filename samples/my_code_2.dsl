main () 
{
	Matrix a = readMatrix ( "../samples/matrixA.data" ) ; 
	Matrix b = readMatrix ( "../samples/matrixB.data" ) ;

	Int rows;
	Int cols;
	rows = numRows(a); 
	cols = numCols(a);
	Int i;
	Int j;
	for(i = 0 : rows - 1)
	{
		for(j = 0 : cols - 1)
		{
			Int currA;
			currA = a[i, j];
			Int currB;
			currB = b[i, j];

			a[i, j] = currA * currB;
			print(a[i, j]);
			print(" ");
		}
		print("\n");
	}
}



