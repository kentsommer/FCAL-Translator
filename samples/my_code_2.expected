5 10 2 15 40 14 20 6 72 60 
3 0 232 64 63 74 174 34 72 90 
3 76 224 27 12 4 0 45 9 72 
1 25 4 20 9 15 81 0 171 16 
0 116 45 84 340 2 245 8 8 14 
0 59 9 20 78 0 9 4 35 12 
4 270 27 248 36 14 8 48 8 27 
2 12 0 1140 0 180 27 49 0 8 
4 16 9 40 90 45 8 8 21 7 
2 16 18 8 672 56 8 18 72 4 
